libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/barChart/PluggableBarChart.tsx

```diff
@@ -148,24 +149,24 @@ export class PluggableBarChart extends PluggableColumnBarCharts {
         }
 
         return {
-            currentSort: [],
+            defaultSort: [],
             availableSorts: [],
         };
     }
     public getSortConfig(referencePoint: IReferencePoint): Promise<ISortConfig> {
-        const { buckets } = referencePoint;
+        const { buckets, properties } = referencePoint;
         const viewBy = getBucketItems(buckets, BucketNames.VIEW);
         const measures = getBucketItems(buckets, BucketNames.MEASURES);
         const disabled = viewBy.length < 1 || measures.length < 1;
-        const { currentSort } = this.getDefaultAndAvailableSort(
+        const { defaultSort, availableSorts } = this.getDefaultAndAvailableSort(
             referencePoint,
-            canSortStackTotalValue(referencePoint.buckets, referencePoint.properties),
+            canSortStackTotalValue(buckets, properties),
```

Comment:
At this point you can just delete this attribute and resolve it in `getDefaultAndAvailableSort` as you have `referencePoint` there. You just need to use a different name for the variable so ti does not clash with the method name.